app/client/src/app/App.js

```diff
@@ -70,7 +73,9 @@ function App() {
         <Route exact path="/" component={LoadableHome} />
         <Route path="/generator" component={LoadableGenerator} />
         <Route path="/about" component={LoadableAbout} />
-        <Route path="*" render={() => <h1>ono 404</h1>} />
+        <Route path="*" component={Page400} />
+        <Route path="*" component={Page404} />
+        <Route path="*" component={Page500} />
```

Comment:
Using the `*` with something like `<Route path="*" component={MyComponent}>` means that `MyComponent` will render for every other route that isn't covered above, so there shouldn't be multiple of them. This is typically used only for the `Page404` component which covers all situations when the user tries to go to an invalid route. So you can get rid of the `Page400` and `Page500` stuff, error-specific pages would probably need to be handled in my `nginx` server setup which I'm not too worried about right now.